libs/ui/src/lib/detailed-status/detailed-status.component.scss

```diff
@@ -0,0 +1,45 @@
+@import '~@angular/material/theming';
+@include mat-core();
+
+$actions-palette: mat-palette($mat-grey, 100);
+$active-button-palette: mat-palette($mat-blue);
```

Comment:
We already have a primary/error palette defined at the application level. Could you import those rather than redefining a slight variation of our primary palette?